course5/homework2.2/homework2.2.cc

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+#include <cmath>
+#include <cstring>
+#include <vector>
+
+static const float EPSILON = 1.0e-8;
```

Comment:
this constant is local to a function, should be defined inside of the function, to not pollute the namespace.